content/blog/2020-07-10-july-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,150 @@
+---
+title: July '20 Heartbeat
+date: 2020-07-10
+description: |
+  Catch up on new DVC releases, talks, and projects in our community. 
```

Comment:
let's wrap it at 80 symbols ... btw, I think only one is required (probably descriptionLong?) if you don't want to make them different. (to avoid duplication)